packages/mint-components/src/components/sqm-country-field/sqm-country-field.tsx

```diff
@@ -0,0 +1,85 @@
+import { isDemo } from "@saasquatch/component-boilerplate";
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, h, Prop, State } from "@stencil/core";
+import deepmerge from "deepmerge";
+import { DemoData } from "../../global/demo";
+import { getProps } from "../../utils/utils";
+import { useDropdownField } from "../sqm-dropdown-field/useDropdownField";
+import {
+  CountryFieldView,
+  CountryFieldViewProps,
+} from "./sqm-country-field-view";
+import { useCountryField } from "./useCountryField";
+
+/**
+ * @uiName Country Field
+ */
+@Component({
+  tag: "sqm-country-field",
+})
+export class CountryField {
+  @State()
+  ignored = true;
+
+  /**
+   * @uiName Country name attribute
+   */
+  @Prop() dropdownName: string = "countryCode";
```

Comment:
This should be `country` not countryCode

Reply:
This is because the managed Identity service transforms formData.country to countryCode on the user upsert. But there is the question on if we should change/follow the default, since `countryCode` as the dropdownName is a bit more correct